app/src/androidTest/java/ca/marshallasch/veil/database/DatabaseTest.java

```diff
@@ -22,13 +22,13 @@
 
     private Database database;
     @Before
-    public void setUp() throws Exception
+    public void setUp()
```

Comment:
Why was the exception removed?

Reply:
Because android studio says it was unused.